File: RAI/EjemploBanco/JavaImplementation/Consumo.java
public class Consumo extends Credito{
    private double morasidad;

    public Consumo(int id, double interes, int monto, int cuotas, double morasidad){
        super(id, interes, monto, cuotas);
        this.morasidad = morasidad;
    }


    public String toString(){
        return super.toString() + "Morosidad: " + this.morasidad;
    }
}

File: RAI/EjemploBanco/python_implementation/Consumo.py
from Credito import Credito

class Consumo(Credito):

    def __init__(self, identificador, interes, monto, cuotas, morasidad):
        super().__init__(identificador, interes, monto, cuotas)
        self.morasidad = morasidad

    def __str__(self):
        return super().__str__() + " Morasidad: " + str(self.morasidad)